src/common/models/ResponsiveImage.ts

```diff
@@ -71,6 +71,24 @@ export const IMAGE_DIMENSIONS: ResponsiveImageDimensions = {
     sm: [390, 468],
     xs: [260, 312],
   },
+  resource: {
+    thumb: DEFAULT_THUMB_SIZE,
+    original: [Infinity, Infinity],
+    wide: [710, 710],
+    lg: [710, 710],
+    md: [710, 710],
+    sm: [540, 540],
+    xs: [360, 360],
+  },
+  hobby: {
+    thumb: DEFAULT_THUMB_SIZE,
+    original: [Infinity, Infinity],
+    wide: [710, 710],
+    lg: [710, 710],
+    md: [710, 710],
+    sm: [540, 540],
+    xs: [360, 360],
+  },
```

Comment:
It is sort of unnecessary to have to identical image-sizes, with different names. I would prefer if we just use `resource` for both